### PR TITLE
Mosaicism

### PR DESCRIPTION
**Pull request by moonso**
- Adds a couple of mosaicism options on variant level
- Highlight mosaic variants in variants view
- Show mosaicism on pinned variants

NOT IMPLEMENTED
- Display mosaic variants in case report
- Show is variant has been marked mosaic in other cases

### CODE REVIEW

**Comment by vwirta** [comment]
How do we identify a variant as mosaic? What caller do we use for this right now?

**Comment by northwestwitch** [comment]
Hi, we don't call the mosaicism, but users can assign this tag manually at the variant page, after validation I guess..

**Comment by henrikstranneheim** [comment]
Usually, _de novo_ variants and manual inspection in IGV and then Sanger

File: scout/server/blueprints/variants/views.py [hunk 1]
```diff
@@ -322,19 +322,31 @@ def variant_update(institute_id, case_name, variant_id):
...
         flash("Reset variant dismissal: {}".format(variant_obj.get('dismiss_variant')), 'info')
 
+    mosaic_tags = request.form.getlist('mosaic_tags')
+    if mosaic_tags:
+        store.update_mosaic_tags(institute_obj, case_obj, user_obj, link, variant_obj,
+                                     mosaic_tags)
+        if new_dismiss:
+            flash("Added mosaic tags: {}".format(mosaic_tags), 'info')
+
+    if variant_obj.get('mosaic_tags') and not mosaic_tags:
```
Comment (dnil) [inline scout/server/blueprints/variants/views.py]: ..and this little shortcut won't work gracefully with multiple save buttons. I'll get back with a fix..

**Comment by dnil** [comment]
Ok, so here goes a rough fix for the multiple buttons cross-resetting issue. Sorry to barge in - HTH!

**Comment by moonso** [comment]
Thank you @dnil ! So I have updated the branch and rebased on master.
Regarding your comments @northwestwitch 
1. I have moved the tag to inheritance column, good suggestion!
2. I don't agree with this comment. Just like the dismiss variant options this will show in the select tags field. That is fine I think

**Comment by dnil** [comment]
I'll assume you have tested setting and resetting different user tags after merge! Could you also write a line or two in the documentation for the users that aren't already in the loop on this? Thanks!

**Pull request merged**